tests/test_pedigree.py

```diff
@@ -174,3 +175,153 @@ def test_pedigree_sanity_checks(self):
 
         with pytest.raises(ValueError):
             ped.get_times(individual=ped.individual)
+
+
+def simulate_pedigree(
+    n_founders=8,
```

Comment:
Generally the convention within msprime and tskit is to be a bit more verbose than this, we'd have ``num_founders`` rather than ``n_founders``. Probably no need to give a default here I'd say, easy enough to specify a number of founders.